packages/babel-types/scripts/generators/generateAsserts.js

```diff
@@ -18,7 +18,7 @@ import is from "../../validators/is";
 function assert(type: string, node: Object, opts?: Object): void {
   if (!is(type, node, opts)) {
     throw new Error(
-      \`Expected type "\${type}" with option \${JSON.stringify(opts)}\`,
+      \`Expected type "\${type}" with option \${JSON.stringify(opts)}, but instead got "\${node.type}".\`,
```

Comment:
Can you break this line like in the generated file? Otherwise we will need to `make fix` every time that it is recreated.

Reply:
I can for consistency sake, but the generated files are auto "prettified" via:
https://github.com/babel/babel/blob/master/packages/babel-types/scripts/utils/formatCode.js

Reply:
Ah ok, nevermind then